dashboard/src/components/Filters.js

```diff
@@ -195,32 +238,87 @@ const ValueSelector = ({ field, filterValues, value, onChangeValue, base }) => {
             isClearable={!value}
             onChange={(e) => {
-              if (!e) return setDateComparator(null);
-              setDateComparator(e.value);
-              onChangeValue({ date: value?.date, dateComparator: e.value });
+              if (!e) return setComparator(null);
+              setComparator(e.value);
+              onChangeValue({ date: value?.date, comparator: e.value });
             }}
           />
-        </Col>
-        {value?.dateComparator !== 'unfilled' && (
-          <Col sm={6}>
+        </div>
+        {value?.comparator !== 'unfilled' && (
+          <div className="tw-basis-1/2 tw-px-4">
             <DatePicker
               locale="fr"
               dateFormat="dd/MM/yyyy"
               className="form-control"
               name={name}
               selected={value?.date ? new Date(value?.date) : null}
-              onChange={(date) => onChangeValue({ date, dateComparator })}
+              onChange={(date) => onChangeValue({ date, comparator })}
             />
-          </Col>
+          </div>
         )}
-      </Row>
+      </div>
+    );
+  }
+
+  if (['number'].includes(type)) {
+    return (
+      <div className="-tw-mx-4 tw-flex tw-flex-wrap">
+        <div
+          className={[
+            'tw-px-4',
+            value?.comparator === 'unfilled' ? 'tw-basis-full' : '',
+            value?.comparator === 'between' ? 'tw-basis-5/12' : '',
+            !['unfilled', 'between'].includes(value?.comparator) ? 'tw-basis-1/2' : '',
+          ].join(' ')}>
+          <SelectCustom
+            options={numberOptions}
+            value={numberOptions.find((opt) => opt.value === value?.comparator)}
+            isClearable={!value}
+            onChange={(e) => {
+              if (!e) return setComparator(null);
+              setComparator(e.value);
+              onChangeValue({ number: value?.number, comparator: e.value });
+            }}
+          />
+        </div>
+        {value?.comparator !== 'unfilled' && (
+          <div className={['tw-px-4', value?.comparator === 'between' ? 'tw-basis-3/12' : 'tw-basis-1/2'].join(' ')}>
+            <input
+              name={name}
+              className="tailwindui !tw-mt-0"
+              type="number"
+              value={value?.number || ''}
+              onChange={(e) => {
+                onChangeValue({ number: e.target.value, number2: value?.number2, comparator });
+              }}
+            />
+          </div>
+        )}
+        {value?.comparator === 'between' && (
+          <>
+            {/* we have an input here, just for styling purpose, to have 'et' aligned with number and number2 */}
+            <input className="tailwindui !tw-mt-0 tw-basis-1/12 !tw-border-0 !tw-bg-transparent !tw-shadow-none" disabled defaultValue="et" />
+            <div className="tw-basis-3/12 tw-px-4">
+              <input
+                name={name}
+                className="tailwindui !tw-mt-0"
+                type="number"
```

Comment:
```suggestion
                type="number"
                min="0"
```

En fait ça arrive jamais qu'on veuille mettre des valeurs négatives